docs/GettingStarted.md

```diff
@@ -1102,6 +1154,30 @@ Then you must enable the request queuing feature in the integration handling the
 
 For Rack based applications, see the [documentation](#rack) for details for enabling this feature.
 
+### HTTP request queuing
```

Comment:
How did this end up in this PR? I think this got merged to `0.13-dev` already. Bad rebase?

Reply:
interesting. Fixing....